### PR TITLE
[FIX JENKINS-33246] Add a footnote about ** in setup wizard

### PR DESCRIPTION
**Pull request by kzantow**
Explain what the *\* indicator is during plugin installation in the setup wizard.

image

image

@reviewbybees esp. @daniel-beck

### CODE REVIEW

**Comment by daniel-beck** [comment]
:+1: Shiny

**Comment by slide** [comment]
+1

File: war/src/main/less/pluginSetupWizard.less [hunk 1]
```diff
@@ -544,12 +552,24 @@
...
-            padding-left: .4em;
+            padding-left: 10px;
 
             &:before {
                 content: ' ** ';
             }
         }
+        
+        .dependency-legend {
+            color: #aaa;
```
Comment (daniel-beck) [inline war/src/main/less/pluginSetupWizard.less]: When viewing the actual dialog, this is so difficult to notice it's basically invisible. Why not just make it black so it's more noticeable?

Comment (kzantow) [inline war/src/main/less/pluginSetupWizard.less]: We can change the dependency color and make them match, but it is confusing when it's a different color, when the dependencies are already a different color. @daniel-beck

Comment (daniel-beck) [inline war/src/main/less/pluginSetupWizard.less]: @kzantow If the price to get more contrast here is more contrast for dependencies in the list, seems like an improvement all around.

**Comment by kzantow** [comment]
@daniel-beck darkness factor achieved

**Comment by daniel-beck** [comment]
@kzantow Thanks!

**Pull request merged**